inc/template-tags.php

```diff
@@ -115,42 +117,40 @@ function twenty_twenty_one_entry_meta_footer() {
 				'</span><br>'
 			);
 
-			/* translators: used between list items, there is a space after the comma. */
-			$categories_list = get_the_category_list( __( ', ', 'twentytwentyone' ) );
-			if ( $categories_list ) {
-				printf(
-					/* translators: 1: posted in label, 2: list of categories. */
-					'<span class="cat-links">%1$s %2$s</span>',
-					esc_html__( 'Categorized as', 'twentytwentyone' ),
-					$categories_list // phpcs:ignore WordPress.Security.EscapeOutput
-				);
-			}
+			if ( has_category() || has_tag() ) {
 
-			/* translators: used between list items, there is a space after the comma. */
-			$tags_list = get_the_tag_list( '', __( ', ', 'twentytwentyone' ) );
-			if ( $tags_list ) {
-				printf(
-					/* translators: %1$s: posted in label, %2$s: list of tags. */
-					'<span class="tags-links">%1$s %2$s</span>',
-					esc_html__( 'Tagged', 'twentytwentyone' ),
-					$tags_list // phpcs:ignore WordPress.Security.EscapeOutput
-				);
-			}
-		} elseif ( is_single() ) {
-			// Posted on.
-			twenty_twenty_one_posted_on();
+				echo '<div class="post-taxonomies">';
```

Comment:
Wrapping this changed the design a bit, so that categories & tags are now on separate lines in archive views, is that intentional?

| Trunk | This branch |
|-------|-------------|
| ![Screen Shot 2020-10-09 at 10 39 15 AM](https://user-images.githubusercontent.com/541093/95596715-bf036e80-0a1b-11eb-8375-c791f519ee35.png) | ![Screen Shot 2020-10-09 at 10 31 13 AM](https://user-images.githubusercontent.com/541093/95596734-c3c82280-0a1b-11eb-938f-f58206456215.png) |

Reply:
They should be on the same line.